netlify/functions/product.js

```diff
@@ -0,0 +1,46 @@
+require("dotenv").config();
+const stripe = require("stripe")(process.env.STRIPE_KEY);
+
+const headers = {
+  "Access-Control-Allow-Origin": "*",
+  "Access-Control-Allow-Headers": "Content-Type",
+  "Content-Type": "application/json",
+};
+
+exports.handler = async ({ httpMethod, queryStringParameters }, context, callback) => {
```

Comment:
@vickywane I see that the implementation is now generic and works for any valid Product ID. Therefore the JS function name should match that behavior and be named `getProductInfo` as opposed to the more narrow `getNotificationProduct` naming.

Reply:
Alright. 

I have adjusted the `operation-id` to effect this change.